### ISSUE TITLE
It isn't clear that you can open results in the inspector

### ISSUE THREAD

**Issue by guyguy2001** [BetterThanTomorrow/calva#2620]
(Alt title: Make items in the inspector have the `>` expanding arrow by default)

## The problem
When running a couple of expressions in the repl, the inspector first looks like this:
![image](https://github.com/user-attachments/assets/0a337fb9-6202-48ef-ae28-90df2d1895c6)

Which seemed to me like it's just a weird way to show the repl results history in a textual way, and nothing more.

I think I had to see in the docs / tutorial that you can expand items.

It isn't unreasonable or anything, but for someone who comes to Calva from other programming languages, and not other clojure editors, and isn't used to REPL programming or paredit, everything is already so overwhelming, that simple things like this can be missed.

(A small related "nitpick" - there isn't any space for the `>` at the start, and once I do inspect an item, all of the items move right to add place for the `>` - which is a tad bit disorienting (as all items move both down and right at the same time), and misses the opportunity to have the negative space where the `>` will be, before inspecting the first item)
![image](https://github.com/user-attachments/assets/fd29f906-2755-4454-9ed0-41e92a7fb748)

## Proposed Solution
Make the `>` appear from the get-go, and lazily inspect the element once we click it.
This is what pylance do for their call / type hierarchies:
https://github.com/user-attachments/assets/9370e230-0b11-4d55-9df6-cf92ccb880b3
(I'm sure the delay is for the analysis and not the rendering since on big projects it's _really_ slow for like 3 items).

I don't see a reason not to do this, or if there is a reason for users to want to know what was already inspected (e.g. a huge object that takes 10 seconds to expand on the first time, and the user wants to know they're clicking the immidiate one), there could be a UI representing whether it's cached or not (maybe the `>` could be greyed a bit, or an underline or something), and there could be a setting to control this.

If it isn't possible / in the meantime, shifting the text a bit to the right to leave space for the `>` would also be nice.

**Comment by PEZ** [comment]
Hi! Thanks for the detailed report. I think the suggestion makes sense. When I first created the inspector I wasn't too familiar with its API, but now I do think we can lazily unpack the items like you suggest.

Do you want to have a look at implementing it? Otherwise I will do it pretty soon. Seems important for the reasons you give.

**Comment by guyguy2001** [comment]
I won't be able to get to it for now; I hope I'll be able to take a stab on this (or other stuff I encounter) once the Jewish holidays come around next month.

**Comment by PEZ** [comment]
Right. Let's see if I get at it before those holidays. 😄 Please don't hesitate to holler my way when you have some time for hacking on Calva. I'm easiest to find at the Clojurian slack.
